src/plugins/common/link/lib/link-plugin.js

```diff
@@ -706,6 +715,57 @@ define([
 			this.hrefField.setTemplate('<span><b>{name}</b><br/>{url}</span>');
 			this.hrefField.setObjectTypeFilter( this.objectTypeFilter );
 
+			this.toggleAnchor = Ui.adopt('toggleAnchor', ToggleButton, {
+				tooltip: i18n.t('button.toggleanchor.tooltip'),
+				icon: "aloha-icon aloha-icon-anchor",
+				click: function() {
+					if (that.anchorField.isVisible()) {
+						that.anchorField.clear();
+					} else {
+						that.anchorField.show();
+						that.anchorField.focus();
+					}
+
+					return false;
+				}
+			});
+
+			this.anchorField = Ui.adopt('editAnchor', Text, {
```

Comment:
Introduce configuration option to enable anchor field support (defaults to false)